get-shit-done/workflows/new-project.md

```
@@ -410,25 +410,80 @@ gsd-sdk query commit "docs: initialize project" .planning/PROJECT.md
+      { label: "Modify some settings", description: "Keep defaults, change a few" },
+      { label: "Configure fresh", description: "Walk through all questions from scratch" }
+    ]
+  }
+])
+```
+
+**If "Use as-is":** use the defaults values for config.json and skip directly to **Commit config.json** below.
+
+**If "Modify some settings":** present a multiSelect listing every setting with its current saved value:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)